packages/quantic/cypress/integration/summary/summary-expectations.ts

```diff
@@ -0,0 +1,56 @@
+import {should} from '../common-selectors';
+import {SummarySelectors, SummarySelector} from './summary-selectors';
+
+function summarytExpectations(selector: SummarySelector) {
+  return {
+    displaySummary: (display: boolean) => {
+      selector
+        .text()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the summary`);
+    },
+    displayRange: (display: boolean) => {
+      selector
+        .range()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the range`);
+    },
+    displayTotal: (display: boolean) => {
+      selector
+        .total()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the total results of search`);
+    },
+    displayQuery: (display: boolean) => {
+      selector
+        .query()
+        .should(display ? 'exist' : 'not.exist')
+        .logDetail(`${should(display)} display the query`);
+    },
+    rangeContains: (value: string) => {
+      selector
+        .range()
+        .contains(value)
+        .logDetail(`range value should contain "${value}"`);
+    },
+    totalContains: (value: number) => {
+      selector
+        .total()
+        .invoke('text')
+        .then((text) =>
+          expect(text).to.be.equal(new Intl.NumberFormat().format(value))
+        )
+        .logDetail(`total value should contain "${value}"`);
+    },
```

Comment:
I'd suggest to also format the value in the expectation detail message so it matches the value that is validated in the component.

```suggestion
    totalContains: (value: number) => {
      const formatNumber = (v: number) => new Intl.NumberFormat().format(v);
      selector
        .total()
        .invoke('text')
        .then((text) =>
          expect(text).to.be.equal(formatNumber(value))
        )
        .logDetail(`total value should contain "${formatNumber(value)}"`);
    },
```